doc/parameters/param_list.md

```diff
@@ -120,7 +120,8 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | `<data source>`.marking | true | Whether source should mark in costmap |
 | `<data source>`.clearing | false | Whether source should raytrace clear in costmap |
 | `<data source>`.obstacle_range | 2.5 | Maximum range to mark obstacles in costmap |
-| `<data source>`.raytrace_range | 3.0 | Maximum range to raytrace clear obstacles from costmap |
+| `<data source>`.raytrace_max_range | 3.0 | Maximum range to raytrace clear obstacles from costmap |
+| `<data_source>`.raytrace_min_range | 0.2 | Minimum range to raytrace clear obstacles from costmap |
```

Comment:
We should make this default to 0 so that current behavior persists on default parameters.